mozilla-mobile/focus-android/app/src/main/res/values/strings.xml

```diff
@@ -643,6 +643,48 @@
     <!-- Preference for autoplay -->
     <string name="preference_autoplay">Autoplay</string>
 
+    <!-- Heading for the instructions to allow a permission -->
+    <string name="phone_feature_blocked_intro">To allow it:</string>
+
+    <!-- First step for the allowing a permission -->
+    <string name="phone_feature_blocked_step_settings">1. Go to Android Settings</string>
+
+    <!-- Second step for the allowing a permission -->
+    <string name="phone_feature_blocked_step_permissions"><![CDATA[2. Tap <b>Permissions</b>]]></string>
+
+    <!-- Button label that take the user to the Android App setting -->
+    <string name="phone_feature_go_to_settings">Go to Settings</string>
+
+    <!-- Third step for the allowing a permission (Fore example: Camera) -->
+    <string name="phone_feature_blocked_step_feature"><![CDATA[3. Toggle <b>%1$s</b> to ON]]></string>
```

Comment:
Please always explain in comments what the placeholder will be replaced with
(localizers need context in localization comments to be able to translate correctly, please see [here](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#note-on-localizers) for more info)

Reply:
It will be replaced with "Camera"

Reply:
You actually need to update the localization comment itself in the source code repo - saying something like "Third step for allowing a permission. The placeholder will be updated by "camera""

Reply:
I made a pull request 
https://github.com/mozilla-mobile/focus-android/pull/6869

Reply:
Thanks - that totally works like that